aip-console-tools-core/src/main/java/com/castsoftware/aip/console/tools/core/services/ApplicationServiceImpl.java

```diff
@@ -67,6 +77,212 @@ public ApiInfoDto getAipConsoleApiInfo() {
         return restApiService.getAipConsoleApiInfo();
     }
 
+    @Override
+    public int fastScan(FastScanProperties fastScanProperties) throws JobServiceException, PackagePathInvalidException, UploadException {
+        log.info("Fast-Scan command will manage '{}' source code using a modern workflow: verbose= '{}'"
+                , fastScanProperties.getApplicationName(), fastScanProperties.isVerbose());
+
+        log.info("Fast-Scan args:");
+        log.info(String.format("\tApplication: %s%n\tFile: %s%n\tsleep: %d%n", fastScanProperties.getApplicationName()
+                , fastScanProperties.getFilePath().getAbsolutePath(), fastScanProperties.getSleepDuration()));
+
+        String applicationGuid;
+        try {
+            boolean OnBoardingModeWasOn = isOnboardingSettingsEnabled();
+            if (!OnBoardingModeWasOn) {
```

Comment:
```suggestion
            if (!isOnboardingSettingsEnabled()) {
```
Don't need to assign to a variable if you're not using that value again later (also, wrong case and odd name for your variable ^^" )

Reply:
These lines were removed and has no effect

Reply:
They're not removed, they still show in the PR, so they are there.

Reply:
hmm! wasn't committed